### PR TITLE
Seemran/configuration

### PR DESCRIPTION
**Pull request by seem-ish**
1. Knowledge-Based Configuration book +  chapter 6
2. Journal of Intelligent Manufacturing +  automotive product configuration paper
3. International Configuration Workshop 2017  (CONFWS'17) + IoT configuration with ASP

### CODE REVIEW

File: procs.bib [hunk 1]
```diff
@@ -2247,6 +2254,14 @@ @book{jm64
...
 }
 
+@book{kbc14,
+  title = {Knowledge-Based Configuration: From Research to Business Cases},
+  editor = {A. Felfernig and L. Hotz and C. Bagley and J. Tiihonen},
+  booktitle = {Knowledge-Based Configuration: From Research to Business Cases},
+  publisher = {Elsevier/Morgan Kaufmann},
+  year = {2014}
+}
+
```
Comment (tortinator) [inline procs.bib:2264]: This should go into krr,bib

File: procs.bib [hunk 2]
```diff
@@ -2247,6 +2254,14 @@ @book{jm64
   year = {1991}
 }
 
+@book{kbc14,
+  title = {Knowledge-Based Configuration: From Research to Business Cases},
```
Comment (tortinator) [inline procs.bib:2258]: Please adapt the key

File: procs.bib [hunk 3]
```diff
@@ -742,6 +742,13 @@ @proceedings{comma10
   year = {2010}
 }
 
+@proceedings{confws17,
+  title = {Proceedings of the Nineteenth International Configuration Workshop (CONFWS'17)},
+  editor = {L. Zhang and A. Haag},
```
Comment (tortinator) [inline procs.bib:747]: Is the non-alphabetical order correct?

**Review by tortinator — CHANGES_REQUESTED** [review]
Thank you @seem-ish :1st_place_medal: , just fix the smalish issues , thank you :nerd_face:

File: procs.bib (hunk 3 above)
Comment (seem-ish) [reply procs.bib:747]: Are you referring to the alphabetical ordering of the editor names? If so, then yes it is correct.

File: procs.bib (hunk 1 above)
Comment (seem-ish) [reply procs.bib:2264]: The book is a collection of articles and cases and each chapter has specific authors. 
I previously added a bib entry for ch 6 and used crossref to the book, like in SATHandbook.
Should I remove individual chapter bib entry?

Comment (tortinator) [reply procs.bib:2264]: It depends, you could also cite it as \cite[Chapter 6]{blabla}

But we can also leave it as is, similar to:

@book{lbai,
  title = {Logic-Based Artificial Intelligence},
  editor = {J. Minker},
  address = {Dordrecht},
  booktitle = {Logic-Based Artificial Intelligence},
  publisher = kluwer,
  year = {2000}
}

**Review by tortinator — APPROVED** [review]

**Pull request merged**